Client/Coordinators/Browser/BrowserCoordinator.swift

```diff
@@ -67,6 +68,11 @@ class BrowserCoordinator: BaseCoordinator, LaunchCoordinatorDelegate, BrowserDel
     func didFinishLaunch(from coordinator: LaunchCoordinator) {
         router.dismiss(animated: true, completion: nil)
         remove(child: coordinator)
+
+        // Once launch is done, we check for any saved Route
+        if let savedRoute {
```

Comment:
This look so weird to me. lol

Reply:
Yeah I'm trying to get used to it, and be up-to-date with standards. I feel ambivalent to its usage though, we can discuss it in an eng meeting?